src/Hive.IO/TestVisualizer/Form1.cs

```diff
@@ -0,0 +1,43 @@
+﻿using System;
```

Comment:
I don't fully get it... are you using this project here to load in `EnergyBalancePlots.cs` from `Hive.IO`?

Reply:
yes. the point is to have the feedback loop (compile, run, test) as quick as possible to stay in the zone.

Reply:
(debugging is also easier)